pyigm/igm/igmsightline.py

```diff
@@ -39,7 +39,7 @@ def from_json(cls, jsonfile, **kwargs):
         return slf
 
     @classmethod
-    def from_igmguesses(cls, radec, zem, igmgfile, name=None, **kwargs):
+    def from_igmguesses_old(cls, radec, zem, igmgfile, name=None, **kwargs):
```

Comment:
I think you can remove this method and have this
functionality in the new one by making radec and zem
optional keywords, e.g.

radec=None, zem=None

and then

``` 
if radec is not None:
         blah blah
```